pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>token-macro</artifactId>
       <version>1.10</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java</artifactId>
+      <version>1.3.0</version>
```

Comment:
Surely this should be factored into a library plugin, rather than loaded separately in many places?